tokenserver/scripts/process_account_events.py

```diff
@@ -88,7 +88,15 @@ def process_account_event(config, body):
         body = json.loads(body)
         event = json.loads(body['Message'])
         event_type = event["event"]
-        email = event["uid"]
+        uid = event["uid"]
+        try:
+            email = "%s@%s" % (uid, event["iss"])
```

Comment:
Sorry, I'm not totally sure what you're getting at here; do you mean something like:

```
if "iss" in event:
    email = "%s@%s" % (uid, event["iss"])
else:
    email = uid
```